docs: update test instructions in the README (#334)
Substra/substra: README.md

To run all tests, use the following command:

```sh
make test
```
  

>  Some of the tests require Docker running on your machine before you launch the tests.

Reviewer: ```suggestion
 Some of the tests require Docker running on your machine before running them.
```

I put it as suggestion but know that suggestion are not signed of, so you might need to do the modification on your own of squash the suggested commits 😇
